docs/source/model_doc/dialo_gpt2.rst

```diff
@@ -0,0 +1,29 @@
+DialoGPT
```

Comment:
this file should be named dialogpt.rst, no?

Reply:
(i.e. why the 2?)

Reply:
true!